src/pydvl/value/shapley/gt.py

```diff
@@ -277,56 +281,66 @@ def reducer(
 
     ###########################################################################
     # Solution of the constraint problem with scipy
+    # DISABLED: highs fails to find solutions that SCS finds
```

Comment:
I think we should remove this since the cvxpy approach is more readable and more stable.

Reply:
hmm... but what if we end up deciding to go for a linear system again... I will leave a note of when this was removed, so as to be able to find it in the history.